lookout/style/format/feature_extractor.py

```diff
@@ -337,6 +335,24 @@ def select_features(self, X: numpy.ndarray, y: numpy.ndarray) -> Tuple[numpy.nda
         self._compute_feature_info()
         return X, self.selected_features
 
+    def get_composite_class_representations(self) -> List[str]:
+        """
+        Return the class representations of composite classes.
+
+        :return: Strings representing the composite classes.
+        """
+        return ["".join(CLASS_REPRESENTATIONS[label] for label in labels)
+                for labels in self.labels_to_class_sequences]
+
+    def get_composite_class_printables(self) -> List[str]:
```

Comment:
same

Reply:
Oh no this doesn't take arguments... OK, I will convert to properties